cdap-app-fabric/src/test/java/co/cask/cdap/gateway/handlers/AuthorizationHandlerTest.java

```diff
@@ -39,6 +39,7 @@
 import co.cask.cdap.security.authorization.AuthorizationContextFactory;
 import co.cask.cdap.security.authorization.AuthorizerInstantiator;
 import co.cask.cdap.security.authorization.DefaultAuthorizationContext;
+import co.cask.cdap.security.authorization.InMemoryAuthorizer;
```

Comment:
no, I moved this class from ``cdap-app-fabric/src/main/test`` to ``cdap-security/src/main/test``. That's why we need the import now.